.github/ISSUE_TEMPLATE/benchmarking_internal.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Benchmarking
+about: System benchmarking task
+title: "[BENCHMARK]"
+labels: benchmark
+assignees: ''
+
+---
+
+### Describe the benchmarking experiment/task
+A clear and concise description of what the task is. If it is an experiment please provide the experimental design.
+
+### Hypothesis/expected behavior or outcome
+A clear and concise description of what you expected to happen.
+
+### Definition of done
+A precise outline for the experiment or task to be considered complete.
+
+### Mandatory checklist before making a PR
+* [ ] The success criteria laid down in “Definition of done” are met.
+* [ ] Experiment is documented - hyperparameters, plots, conclusions/findings etc.
+* [ ] Documentation is updated - README, CONTRIBUTING, or other documentation.
+* [ ] Link experiment/benchmarking.
```

Comment:
This last point should maybe be optional. I am not sure if we can make experiment logs public according to our current setup.

Reply:
Done